hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/node/TestSCMNodeManager.java

```diff
@@ -465,6 +465,18 @@ private void assertPipelineCreationFailsWithNotEnoughNodes(
         actualNodeCount);
   }
 
+  private void assertPipelineCreationFailsWithExceedingLimit(int limit) {
+    SCMException ex = assertThrows(SCMException.class, () -> {
+      ReplicationConfig ratisThree =
+          ReplicationConfig.fromProtoTypeAndFactor(
+              HddsProtos.ReplicationType.RATIS,
+              HddsProtos.ReplicationFactor.THREE);
+      scm.getPipelineManager().createPipeline(ratisThree);
+    }, "3 nodes should not have been found for a pipeline.");
```

Comment:
nit: Get replication config outside of `assertThrows`, then block lambda can be simplified to statement.